src/main/java/rails/game/action/BuyCertificate.java

```diff
@@ -41,6 +41,10 @@
     protected int price;
     protected int maximumNumber;
 
+    // In exceptional cases, a president's share may end up in the pool and be buyable.
+    // This can occur after a bankruptcy in 1835 and 18Scan
+    protected boolean president = false;
```

Comment:
If the field can be accessed by a setter and getter method I think it should be made private

Reply:
I would prefer to give subclasses direct access.
Unfortunately, Java has no keyword for such access that excludes other classes in the same package.